.github/workflows/release.yaml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   goreleaser:
-    timeout-minutes: ${{ fromJSON(vars.GHA_DEFAULT_TIMEOUT) }}
```

Comment:
Is this var common across org or repo?
Approving to unblock release - IMO better to set it as repo level variable

Reply:
This is an org level var